### DISCUSSION TITLE
Working on preparing version 3.1

### DISCUSSION THREAD

**Discussion by jarlah** [freeacs/freeacs#385] [Announcements]
This repository has been dead for a while, and was marked as deprecated, but the value of the acs server is still there and I decided to bring it up to modern standards and make sure it will never fall back again.

Help or input is greatly appreciated

**Comment by danp86** [comment]
Hello, could you tell me where I can modify the access credentials of the CPEs to the TR-069 service? I can't figure out where they are located.

You're welcome, thank you for your work.

**Reply by jarlah** [reply]
I had basically abandoned this project then revived it only for the sake of trying to bring this dead corpse of a project  back up to standards. But it seems like an impossible feat. sorry no support.
